languages/en-US/general/fortnite.json

```
@@ -2,9 +2,9 @@
     "DESCRIPTION": "Shows a player's Fortnite stats!",
     "USAGE": "{{prefix}}fortnite [psn/xbl/pc] [pseudo]",
     "EXAMPLES": "{{prefix}}fortnite pc Ninja",
-    "MISSING_PLATFORM": "Enter a valid platform: `psn`, `pc` or `xbl`!",
-    "MISSING_USERNAME": "Enter a valid epic games username!",
-    "NOT_FOUND": "Player `{{search}}` not found on the platform `{{platform}}`!",
+    "MISSING_PLATFORM": "Please specify platform between `psn`, `pc` or `xbl`!",
```

Sauf erreur de ma part, ce `a` doit être là ? 
```suggestion
    "MISSING_PLATFORM": "Please specify a platform between `psn`, `pc` or `xbl`!",
```